godel_robots/abb/abb_file_suite/src/ftp_upload.cpp

```diff
@@ -0,0 +1,139 @@
+#include "ftp_upload.h"
+
+#include <stdlib.h>
+#include <stdio.h>
+
+#include <curl/curl.h>
+
+/* parse headers for Content-Length */
+static size_t getcontentlengthfunc(void *ptr, size_t size, size_t nmemb, void *stream)
+{
+  int r;
```

Comment:
This code is pretty much all copied from libcurl's FTP upload w/ resume example [here](https://github.com/bagder/curl/blob/a47137c4a5a12c8f8a2626e50116e893e6852b24/docs/examples/ftpuploadresume.c). For whatever reason their main sight is down or showing as jibberish to me.

Would it be okay to make mention of this fact at the top of the source file, and let the internal ROS naming incongruities be? The external API in the header is standards compliant.